vivarium/experiments/store_api.py

```diff
@@ -36,25 +37,27 @@ def test_insert_process() -> Store:
     assert isinstance(
         store['process3'].value, Process), \
         'process3 not inserted successfully'
-    return store
+    if return_value:
+        return store
+    return None
 
 
 def test_rewire_ports() -> None:
     """connect a process' ports to different store"""
-    store = test_insert_process()
+    store = cast(Store, test_insert_process(return_value=True))
```

Comment:
what is this fixing?

Reply:
The cast fixes a `mypy` error. `test_insert_process` now returns either a `Store` or `None`, and `mypy` is not smart enough to realize on its own that this particular return value is a `Store`. It complains that we try to index into this value (e.g. `store[index]`) later on.

Reply:
sounds good to me!